gcloud-java-storage/src/main/java/com/google/gcloud/storage/Blob.java

```diff
@@ -256,4 +261,87 @@ public URL signUrl(long expirationTimeInSeconds, SignUrlOption... options) {
   public Storage storage() {
     return storage;
   }
+
+  /**
+   * Gets the requested blobs. If {@code infos.length == 0} an empty list is returned. If
+   * {@code infos.length > 1} a batch request is used to fetch blobs.
+   *
+   * @param storage the storage service used to issue the request
+   * @param infos the blobs to get
+   * @return an immutable list of {@code Blob} objects. If a blob does not exist or access to it has
+   * been denied the corresponding item in the list is {@code null}.
+   */
+  public static List<Blob> get(final Storage storage, BlobInfo... infos) {
+    checkNotNull(storage);
+    checkNotNull(infos);
+    int length = infos.length;
+    switch (length) {
+      case 0:
+        return Collections.emptyList();
+      case 1:
+        return Collections.singletonList(
+            new Blob(storage, storage.get(infos[0].bucket(), infos[0].name())));
```

Comment:
I like this optimization but I think it is in-consistent with the list (as I think a single get returns null for NOT_FOUND, but
does throw the exception for ACL). A change in either direction is fine with me, but maybe a more desirable one is to leave the single get as is (so, one has a way to get a detail on a failure if desired by using that method).